src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -377,4 +424,4 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	};
 
 	$( obj.ready );
-})( jQuery, tribe.tickets.commerce, Stripe, ky );
+})( jQuery, tribe.tickets.commerce, Stripe, tribe.ky );
```

Comment:
The object here should be `tribe.tickets.commerce.gateway.stripe.checkout`

Reply:
@juanfra this fix was edded to #2264